app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt

```diff
@@ -93,6 +101,41 @@ class AddDeadlineActivity : AppCompatActivity() {
         textTime.text = selectedDate.toLocalTime().toString()
     }
 
+    /**
+     * Setup the PlacesAutocompleteClient used to select a location for a deadline
+     */
+    private fun initializePlacesAutocomplete() {
+        Places.initialize(applicationContext, getString(R.string.places_key))
+        // Must be commented for now
+        /*
+        Places.createClient(this)
+        */
+        initializeAutocompleteFragment()
+    }
+
+    /**
+     * Setup the AutocompleteSupportFragment with all its parameters
+     */
+    private fun initializeAutocompleteFragment() {
+        val autocompleteFragment =
+            supportFragmentManager.findFragmentById(R.id.autocomplete_fragment) as AutocompleteSupportFragment
+
+        autocompleteFragment
+            .setPlaceFields(listOf(Place.Field.ID, Place.Field.NAME)) //Returned fields
+            .setCountries(listOf("CH")) // Default startup country
+
+        autocompleteFragment.setOnPlaceSelectedListener(object : PlaceSelectionListener {
+            override fun onPlaceSelected(place: Place) {
+                // TO DO: Define here what we do with the returned place
+                Log.i(ContentValues.TAG, "Place: ${place.name}, ${place.id}")
+            }
+
+            override fun onError(status: Status) {
+                Log.i(ContentValues.TAG, "An error occurred: $status")
```

Comment:
If that is really an error (in the sense of something that should normally not happen), it would be better to use another level of logging. IIRC `Log.i` means "information", probably another function (`Log.e`?) is more suited to reporting an error